### PR TITLE
Modify bind pieces endpoint to accept holdingId and locationId

### PR DESCRIPTION
**Pull request by Saba-Zedginidze-EPAM**
## Purpose
Modify bind pieces endpoint to accept holdingId and locationId

## Approach
* Add holdingId and locationId to bindPiecesCollection.json

### CODE REVIEW

File: mod-orders/schemas/bindPiecesCollection.json [hunk 1]
```diff
@@ -30,8 +30,13 @@
...
         },
-        "permanentLocationId": {
-          "description": "UUID of the permanent location Id",
+        "locationId": {
+          "description": "UUID of the location Id",
+          "type": "string",
+          "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+        },
+        "holdingId": {
+          "description": "UUID of the holding Id",
```
Comment (SerhiiNosko) [inline mod-orders/schemas/bindPiecesCollection.json:39]: sample files also should be updated

File: mod-orders/examples/bindPiecesCollection.sample [hunk 2]
```diff
@@ -5,7 +5,8 @@
     "materialTypeId": "a058cb27-a77d-4cb3-b76b-2adc931fb0c9",
     "callNumber": "BF2050 .M335 2000",
     "permanentLoanTypeId": "41673a86-ebab-4a5a-8b1b-76110a6f86f4",
-    "permanentLocationId": "02cc63e9-dda0-4b49-bbc6-6253edffcc67",
+    "locationId": "02cc63e9-dda0-4b49-bbc6-6253edffcc67",
+    "holdingId": "0804ddec-6545-404a-b54d-a693f505681d",
```
Comment (SerhiiNosko) [inline mod-orders/examples/bindPiecesCollection.sample:9]: only locationId or holdingId can be presented, but not both at the same time

Comment (Saba-Zedginidze-EPAM) [reply mod-orders/examples/bindPiecesCollection.sample:9]: Is there any way to have two sample files? Or only one is supported to be displayed?

**Review by azizbekxm — APPROVED** [review]

File: mod-orders/examples/bindPiecesCollection.sample (hunk 2 above)
Comment (SerhiiNosko) [reply mod-orders/examples/bindPiecesCollection.sample:8]: you can remain for example holdingId and it is enough

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**